src/digraph.rs

```diff
@@ -1076,6 +1076,61 @@ impl PyDiGraph {
         Ok(out_map)
     }
 
+    /// Get the neighbors of a node.
```

Comment:
LGTM. Just one comment. I think we should note that PyDiGraph.neigbors is the same as PyDiGraph.successor_indices (and different from PyDiGraph.to_directed().neighbors). 
E.g. Get the neighbors (i.e. successors) of a node.

Reply:
That's a good point, added in: https://github.com/Qiskit/retworkx/pull/147/commits/37c8785860b82120fd3d8dc6eff01fb80f4be4e4